downloads.markdown

```diff
@@ -6,48 +6,40 @@ isDownloads: true
+### What you get
 
-* * *
+*   The [Glasgow Haskell Compiler](https://www.haskell.org/ghc)
+*   The [Cabal build system](https://www.haskell.org/cabal/), which can install new packages, and by default fetches from [Hackage](https://hackage.haskell.org/), the central Haskell package repository.
+*   the [Stack](http://docs.haskellstack.org) tool for developing projects
+*   Support for profiling and code coverage analysis
+*   35 core & widely-used [packages](https://www.haskell.org/platform/contents.html)
 
-## Minimal installers
+### How to get it
 
-### What they are<a name="minimal"></a>
+The Platform is provided as a single installer, and can be downloaded at the links below.
 
-Minimal installers provide centrally the [GHC](https://www.haskell.org/ghc) compiler and the [Cabal](https://www.haskell.org/cabal/) and [Stack](https://github.com/commercialhaskell/stack) tools for installing packages. Some may install further build tools (i.e. for parsing and lexing) as well.
+*   [Linux](http://www.haskell.org/platform/linux.html)
+*   [OS X](http://www.haskell.org/platform/mac.html)
+*   [Windows](http://www.haskell.org/platform/windows.html)
 
-### What you get
+* * *
 
-*   Only the core libraries necessary for each platform are included.
-*   Cabal or Stack must be used to download and install packages after installation.
+### Haskell IDEs & other distributions
 
-### How to get them
+In addition to the generic, cross-platform Haskell toolchain described above, there are also easy-to-use, platform-specific distributions and IDEs. The Haskell Wiki contains a [list of the most popular ones](https://wiki.haskell.org/Distributions).
 
-*   [Linux](/downloads/linux)
-*   [OS X](https://www.haskell.org/platform/mac.html) (via the core platform)
-*   [Windows](https://www.haskell.org/platform/windows.html) (via the core platform)
 
-### Where to get help<a name="help"></a>
+* * *
 
-*   For help learning Haskell itself, start with the [Documentation](https://www.haskell.org/documentation) page on the [Haskell Wiki](https://wiki.haskell.org/).
-*   If you need help with [GHC](https://www.haskell.org/ghc)---the Haskell compiler---there is a comprehensive [GHC User Manual](https://downloads.haskell.org/~ghc/latest/docs/html/users_guide/index.html).
-*   For help using Cabal to download or create additional packages (see [below](#libraries)), there is the [Cabal User Guide](https://www.haskell.org/cabal/users-guide/).
-*   For help using Stack to download or create packages, see the stack documentation [below](#stackhelp).
-*   Finally, you can ask questions of other Haskell users and experts on the [#haskell IRC channel](irc://irc.freenode.net/haskell) on the Freenode IRC network.
+## Advanced Installation with Stack
```

Comment:
IMO, stack doesn't belong here. It belong to the wiki page: https://wiki.haskell.org/Distributions